helm/defectdojo/values.yaml

```diff
@@ -233,6 +233,14 @@ django:
   # @type: array<map>
   extraVolumes: []
 
+  # This feature needs more preparation before can be enabled, please visit KUBERNETES.md#media-persistent-volume
+  mediaPersistentVolume:
+    enabled: true
+    name: media
+    type: emptyDir
+    # could be emptyDir (not for production) or pvc
```

Comment:
if the comment is related to the line type: emptyDir, please place the comment above.